fighters/gamewatch/src/acmd/specials.rs

```diff
@@ -1,11 +1,13 @@
-
 use super::*;
 
 #[acmd_script( agent = "gamewatch", scripts = ["game_specialn", "game_specialairn"], category = ACMD_GAME, low_priority)]
 unsafe fn gamewatch_special_n_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
+    frame(lua_state, 1.0);
+    FT_MOTION_RATE_RANGE(fighter, 1.0, 18.0, 9.0);
     frame(lua_state, 18.0);
+    FT_MOTION_RATE(fighter, 1.0);
```

Comment:
not listed in changelog?

Reply:
This was motion rated in opff previously, this change brings it into acmd and cleans it up